numpy/fft/fftpack.py

```diff
@@ -42,6 +42,11 @@
 _fft_cache = {}
 _real_fft_cache = {}
 
+
+def _asarray_copy(*args, **kwargs):
```

Comment:
Might as well use `np.array(..., dtype=complex)`, sprinkle in the `copy=True` if you like (but it is default).